cmake/onnxruntime_training.cmake

```diff
@@ -280,4 +280,4 @@ if (onnxruntime_BUILD_UNIT_TESTS)
     set_target_properties(onnxruntime_training_api_test_runner PROPERTIES FOLDER "ONNXRuntimeTest")
   endif()
 
-endif()
+endif()
```

Comment:
nit: remove this unnecessarily change.

Reply:
this is a formatter run change, prefer to keep it anyway